pages/index.vue

```diff
@@ -27,5 +27,15 @@ appspot.com/storage_companies%2FSony%2FLogo%2Fsony.png`,
       }"
       class="my-5 d-flex justify-center"
     />
+
+    <CompanyGrid>
+      <CompanyCard
+        :content="{
+          logo: `https://1000logos.net/wp-content/
+          uploads/2021/05/Sony-logo.png`,
```

Comment:
Put this on one line, image won't render otherwise.

Reply:
this is just a test case, not the real code? was thinking of removing it before merging

Reply:
actually, I can't put it on a single line, get a lint error. CompanyHero has the same solution as me, and it renders fine. Tested